### PR TITLE
FEATURE: MSF-22578,MSF-22133 Add Gooddata platform parameter snowflake and set TLS default mysql

### PR DESCRIPTION
**Pull request by tqtu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/999/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/886/ : SUCCESS in 26m 38s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1001/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/888/ : FAILURE in 22m 35s

File: lib/gooddata/cloud_resources/snowflake/snowflake_client.rb [hunk 1]
```diff
@@ -77,7 +80,21 @@ def connect
         prop.setProperty('warehouse', @warehouse)
         prop.setProperty('db', @database)
```
Comment (danh-ung) [inline lib/gooddata/cloud_resources/snowflake/snowflake_client.rb:81]: Could we simple set prop.setProperty('application', 'GoodData_Platform') ?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1002/ : SUCCESS in 21s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/889/ : SUCCESS in 21m 41s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1007/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/894/ : SUCCESS in 19m 52s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ruby-gate-PR1821

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-promote-pipeline/167/ : SUCCESS in 20m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-sonarqube-scan-zuul-gate/58/ : SUCCESS in 2m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/600/ : SUCCESS in 47s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-gitops-deploy-pipeline/24/ : SUCCESS in 1m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-fossa-scanning-zuul-el7/143/ : FAILURE in 3m 53s (non-voting)

**Pull request merged**